src/Sulu/Bundle/MediaBundle/Tests/Unit/Media/ImageConverter/ImagineImageConverterTest.php

```diff
@@ -327,27 +325,24 @@ public function testSupportedOutputFormatsWithSimpleValidMimeType(string $mimeTy
         $this->assertEquals($result, $this->imagineImageConverter->getSupportedOutputImageFormats($mimeType));
     }
 
-    public function getSimpleExtensionsByMimeTypes(): \Generator
+    public static function getSimpleExtensionsByMimeTypes(): \Generator
```

Comment:
test method changes should be done on 2.5 branch if possible and non breaking changes